kotlin/add-graphics-renderer/README.metadata.json

```
@@ -0,0 +1,30 @@
+{
+    "category": "Visualization",
+    "description": A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only effect graphics that do not specify their own symbol style.,
```

missing quotation marks from the description
```suggestion
    "description": "A renderer allows you to change the style of all graphics in a graphics overlay by referencing a single symbol style. A renderer will only effect graphics that do not specify their own symbol style.",
```